puppet/modules/web/manifests/vhost/yum.pp

```diff
@@ -27,11 +29,21 @@
     },
   ]
 
+  if $setup_receiver {
+    secure_ssh::receiver_setup { $user:
+      user           => $user,
+      foreman_search => 'host ~ node*.jenkins.osuosl.theforeman.org and (name = external_ip4 or name = external_ip6)',
+      script_content => file('web/deploy-yumrepo.sh'),
+    }
+  }
+
   web::vhost { 'yum':
-    servername   => $servername,
-    docroot      => $yum_directory,
-    docroot_mode => '2575',
-    directories  => $yum_directory_config,
+    servername    => $servername,
+    docroot       => $yum_directory,
```

Comment:
don't we need to create that? or will web::host (or apache::vhost) handle that for us?

Reply:
apache::vhost will, which is why we pass the owner, group and mode.

Reply:
cool.